salesagility/SuiteCRM
# WIP: Backport php-cs-fixer to hotfix-7.10.x and add linting to Travis CI

## Description
NOTE: This depends on the changes from #7471 before it can be merged. php-cs-fixer hasn't supported PHP 5.5 since v2.2.x in 2017, so it can't be used without dropping support for PHP 5.5.

This does a few things:
- Adds php-cs-fixer to the composer.json as a dev dependency. (backport of #7206)
- Backports the php-cs-fixer Robo commands from hotfix.
- Fixes all linting failures that were present in the codebase.
- Adds a job to run php-cs-fixer in Travis CI.

We may want to move the Lint job up to the top so it fails without needing to wait for all the test suites to run?

## Motivation and Context
If the linter isn't being run in CI, it's not being enforced. And if the linter is only run on hotfix, changes in hotfix-7.10.x aren't linted, and can cause linter failures in hotfix when hotfix-7.10.x is merged in.

## How To Test This
Make sure the linter passes in CI. Also, try doing a `composer install` and running it locally with `./vendor/bin/robo style:phpcsfixer-dry-run`

## Types of changes

### Final checklist
